src/harness/testcases/testdata/esc_sensor_record_0.json

```diff
@@ -1,12 +1,10 @@
 {
   "id": "esc_sensor/admin_0/esc_sensor_0",
- "installationParam":{
-   "latitude": 39.0119,
+  "installationParam": {
+    "latitude": 39.0119,
     "longitude": -98.4842,
     "height": 9.3,
     "heightType": "AGL",
-    "indoorDeployment": false,
-    "antennaGain": 16,
-    "antennaBeamwidth": 30
- }
+    "antennaGain": [9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,14,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,1,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,1,9,10,9,8,10,16,7,8,10,9,8,7,10,16,7,8,9,10,16,8,9,10,9,8,7,9,8,10,16,12,12,16,19,10,9,7,9,1,7,9,10,12,9,7,6,9,10,16,12,10,14,14,10,12,10,7]
```

Comment:
Hmm, does this mean that we need to change the SAS-SAS TS? It refers to the SAS-CBSD TS, which says this field should be a scalar. Can you start a thread with (at least) Greg Billock and myself?